deep_qa/layers/entailment_models/tuple_match.py

```diff
@@ -34,31 +34,52 @@ class TupleMatch(Layer):
     ----------
     - num_hidden_layers : int, default=1
         Number of hidden layers in the shallow NN.
+
     - hidden_layer_width : int, default=4
         The number of nodes in each of the NN hidden layers.
+
     - initialization : string, default='glorot_uniform'
         The initialization of the NN weights
+
     - hidden_layer_activation : string, default='relu'
         The activation of the NN hidden layers
+
     - final_activation : string, default='sigmoid'
         The activation of the NN output layer
+
+    - name : string, default='tuple_match'
```

Comment:
This one is already covered by the super class, so you don't need to have it here.